openmc_plotter/main_window.py

```diff
@@ -1065,6 +1077,18 @@ def restoreModelSettings(self):
             with open('plot_settings.pkl', 'rb') as file:
                 model = pickle.load(file)
 
+                # check if loaded cell/mat ids hash match the pkl file:
+                current_mat_xml_hash = hash_file('materials.xml')
+                current_geom_xml_hash = hash_file('geometry.xml')
+                if (current_mat_xml_hash != model.mat_xml_hash) or \
+                    (current_geom_xml_hash != model.geom_xml_hash):
+                    # hashes do not match so ignore plot_settings.pkl file
+                    pkl_settings_warn = "WARNING: Model has changed since " +\
+                                        "storing plot settings. Ignoring " +\
+                                        "previous plot settings."
+                    print(pkl_settings_warn)
```

Comment:
Let's use `warnings.warn` here to make the message more apparent to the users. Ideally, we'd also pop a window to make this clear to the user in case they aren't paying attention to the terminal (I'm frequently guilty of that w/ other applications).

An example of this can be found when trying to open a new statepoint when one is already open in the plotter -- https://github.com/openmc-dev/plotter/blob/a8abcda71c20f6f51f635636338a3ee95a78733e/openmc_plotter/main_window.py#L519